linked_list/linked_list.py

```diff
@@ -0,0 +1,203 @@
+from __future__ import print_function  # For Python 2
+
+
+class node(object):
+    def __init__(self, data=None, next_=None):
```

Comment:
There is no need to use `_` after the variable name. `self.next` and `next` will be different.